pages/user guides/importing_to_excel.md

```diff
@@ -34,42 +34,133 @@ Ie. If your Application has a dependency on a server, you need to have that serv
-If this is the case you should do this before you import your data.
+There are several default columns values included for each record type, 
+however if you have additional information on your inventory that would like to include, you can create custom fields for a record (Check step 3). 
+Remember to add those fields before importing the data.
 
-## Importing your data
+## How to import your data
 
-- Select the data type you wish to import.
+1. Select the data type you wish to import.
 
-{% include image.html file="step0.PNG" %}
+    {% include image.html file="step0.PNG" %}
 
-- Select the sheet in your file that you want to import data from.
+2. Select the sheet in your file that you want to import data from.
 
-{% include image.html file="step1.PNG" %}
+    {% include image.html file="step1.PNG" %}
 
-- Map the columns of your sheet to the fields from the drop down. If you have a column that is not present as a field and you want to import it, you can add that field.
+3.  Map the columns of your sheet to the fields from the drop down. If you have a column that is not present as a field and you want to import it, you can add that field.
 
-{% include image.html file="step2.PNG" %}
+    {% include image.html file="step2.PNG" %}
 
-- Click submit to import the data.
+4. Click submit to import the data.
 
-{% include note.html content="If you are importing applications, the name field is required for each application in order to successfully import. For any of the other record types you are importing, there are no fields that are required in order for you to import them." %}
+{% include note.html content="If you are importing applications, the _name_ field is required for each application in order to successfully import. For any of the other record types you are importing, there are no fields that are required in order for you to import them." %}
 
 {% include tip.html content="The Tags option can be used to generate multiple tags and associate them with the record. You can use this track any type of categorical data. Additionally you can search and filter for records based on tags. Multiple tags can be associated based on one column of data, they are parsed and split on any commas (,)" %}
 
 ### Importing Virtualization Clusters
 
-Tidal Migration uses the following fields to map a virtualization cluster to other resources: _Hypervisor Technology, Servers_. For this field, you must select the property that the column will correspond to, generally this is _Name_.
+Default fields:
+- Hypervisor Technology
+- Servers
+
+There is no required field when importing Virtualization clusters. 
+
+* _Hypervisor Technology_, this field is used to list the technologies you have in your inventory. 
+
+* _Servers_, this field is used to create associations as you import the rest of your inventory, in this instance, _Servers_ refers to the server (host name) hosting the hypervisor.
```

Comment:
>this field is used to create associations as you import the rest of your inventory

I don't follow what this means.

> _Servers_ refers to the server (host name) hosting the hypervisor.

This is definitely accurate but i think not clear and maybe ambiguous, regarding 'hosting the hypervisor'. as there are other servers that host the control plane of the hypervisor. while these physical vms in terms of usage, host the virtual servers that apart of the cluster, and they do technically also host part of the hypervisor control plane to do that.

> A list of physical servers that are used to run all the virtual machines within the Cluster. The servers for each Cluster should be provided as a list of hostnames that are comma (,) separated.

Reply:
>this field is used to create associations as you import the rest of your inventory

It took me some time but I get what this first part means now. The part that i was struggling with was _the rest of your inventory. I didn't follow what that referred to.

>As you import other records, (ex. Applications, Databases) this field is used to find the correct Server and create those dependencies.

That's just an example of what, in my case, would have avoided me not knowing at first.

Reply:
Nice, your last suggestion reads a lot better and reduces the extra complexity